common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -13,24 +14,20 @@
 
 namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
 {
-#pragma warning disable CA1001 // Types that own disposable fields should be disposable - WS is owned by the caller.
-
+    //[SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "WS is owned by the caller.")]
```

Comment:
I have this ready, but will disable it until actually dealing with `CancellationTokenSource`